src/js/user-profile/containers/UserProfileApp.jsx

```diff
@@ -32,10 +32,10 @@ class UserProfileApp extends React.Component {
         <div className="usa-width-two-thirds medium-8 small-12 columns">
           <h1>Your Account</h1>
           <div>
-            {(__BUILDTYPE__ !== 'production') && <FormList
+            <FormList
```

Comment:
Should the tests for UserProfileApp be updated to reflect this? Or are the tests for FormList sufficient and that would that be a case of testing React and not our code?

Reply:
Added a saved form to the mock data and a basic test.